### DISCUSSION TITLE
Create a custom Dataset without labels

### DISCUSSION THREAD

**Discussion by kalikhademi** [open-mmlab/mmclassification#1542] [General]
Hi everyone, 

I checked the code for creating the new dataset but all the examples are for labeled data. I wonder if anyone knows how I should change the python script to create a custom dataset for open images (https://storage.googleapis.com/openimages/web/factsfigures_v6.html).

**Comment by Ezra-Yu** [comment]
Please refer to [this tutorial](https://mmpretrain.readthedocs.io/en/latest/notes/pretrain_custom_dataset.html). This example uses the MAE self-sup algorithm to train a pre-train model.
